stdlib/ballerina-builtin/src/main/ballerina/ballerina/jwt/issuer.bal

```diff
@@ -50,22 +51,22 @@ function createHeader (Header header) returns (string) {
     return urlEncode(util:base64Encode(headerJson.toString()));
 }
 
-function createPayload (Payload payload) returns (string, error) {
+function createPayload (Payload payload) returns (string) | error {
```

Comment:
It would be better to re-arrange the parenthesis like `(string|error)`.

Reply:
this way is correct as well right?